gke-disk-image-builder/script/startup.sh

```diff
@@ -27,16 +27,23 @@ else
   exit 1
 fi
 
+# Check if disk is partitioned and update disk path accordingly if so.
+DEVICE_NODE=/dev/disk/by-id/google-secondary-disk-image-disk
+if [[ -e "$DEVICE_NODE-part1" ]]; then
+  DEVICE_NODE="$DEVICE_NODE-part1"
+fi
+echo "using device node: $DEVICE_NODE"
+
 # Check if the device exists
-if ! [ -b /dev/sdb ]; then
-  echo Device /dev/sdb does not exist. Please rerun the tool to try it again.
+if ! [ -b /dev/disk/by-id/google-secondary-disk-image-disk ]; then
```

Comment:
should we use `$DEVICE_NODE` instead of the absolute path (same below)?

Reply:
thanks, i actually had this locally already but forgot to push the changes! done.